kbi/0013/index.rst

```diff
@@ -0,0 +1,95 @@
+.. index::
+   single: filter-branch; merge; copy-file
+
+.. highlight:: console
+
+KBI0013: Merge Data Availability Information
+============================================
+
+:authors: Laura Waite <laura@waite.eu>
+:discussion: <link>
+:keywords: git-annex-filter-branch, availability info
+:software-versions: datalad_0.18.3, git-annex_10.20230126
+
+Situations can arise when one wishes to split apart an existing large
+dataset into multiple subdatasets. The command `datalad copy-file`_ works very
+well for this when file ability information is URL-based; however, this is not
```

Comment:
Maybe we can leave this link here as a reference for a copy-file example: http://handbook.datalad.org/en/latest/beyond_basics/101-149-copyfile.html

Reply:
Done in 114c9677e655bb2fcecca7d4ac7c9ccaa5747cf8